db/migrate/20221102173636_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reviews do |t|
+      t.string :body, null: false
+      t.string :rating, null: false
+
+      t.belongs_to :trail, null: false
```

Comment:
we should set up reviews to be also associated with a user - you can display the reviews anonymously, but on the DB we should be tracking who made the review. this is common practice in the industry, and will set you up for potential future features such as editing or deleting reviews (which we want to make sure only the user who created that review can edit or delete it!)